grpc/internal/resolver/dns/dns_resolver.go

```diff
@@ -87,8 +87,8 @@ func NewDefaultSRVBuilder() resolver.Builder {
 	return &srvBuilder{scheme: "srv"}
 }
 
-// NewSRVBuilder creates a srvBuilder which is used to factory SRV DNS resolvers
-// with a custom grpc.Balancer use by nonce-service clients.
+// NewNonceSRVBuilder creates a srvBuilder which is used to factory SRV DNS
+// resolvers with a custom grpc.Balancer use by nonce-service clients.
```

Comment:
```suggestion
// resolvers with a custom grpc.Balancer used by nonce-service clients.
```
My typo, not yours, but we should fix it while we're here.